I cannot run the tests at the moment. I've never done that before, so I don't know what the problem might be.

I have a venv set up & activated; it's the one I use for running my changed states/modules. I did read `HACKING.rst` and installed the dev requirements for Python 2.7 in the venv. However, running `./setup.py test` results in:

```
# tons of messages about installing stuff, then:
running test
Traceback (most recent call last):
  File "/home/mosu/prog/salt/tests/runtests.py", line 61, in <module>
    from tests.integration import TestDaemon  # pylint: disable=W0403
  File "/home/mosu/prog/salt/tests/integration/__init__.py", line 34, in <module>
    from tests.support.processes import *  # pylint: disable=wildcard-import
  File "/home/mosu/prog/salt/tests/support/processes.py", line 18, in <module>
    from pytestsalt.utils import collect_child_processes, terminate_process, terminate_process_list  # pylint: disable=unused-import
ImportError: No module named pytestsalt.utils
```

Yeah… there's no file or directory whose name contains `pytestsalt`:

```
[0 mosu@sweet-chili (fix-file-comment) ~/prog/salt] find -iname '*pytestsalt*'
[0 mosu@sweet-chili (fix-file-comment) ~/prog/salt] find ~/opt/salt-dev/ -iname '*pytestsalt*'
[0 mosu@sweet-chili (fix-file-comment) ~/prog/salt]
```

So… what now?

--- a/tests/unit/states/test_file.py
+++ b/tests/unit/states/test_file.py
@@ -1167,7 +1167,7 @@ class TestFileState(TestCase, LoaderModuleMockMixin):
 
             with patch.object(os.path, 'isabs', mock_t):
                 with patch.dict(filestate.__salt__,
-                                {'file.search': MagicMock(side_effect=[True, True, True, False, False])}):
+                                {'file.search': MagicMock(side_effect=[False, True, False, False])}):
                     comt = ('Pattern already commented')
                     ret.update({'comment': comt, 'result': True})
                     self.assertDictEqual(filestate.comment(name, regex), ret)
@@ -1177,7 +1177,7 @@ class TestFileState(TestCase, LoaderModuleMockMixin):
                     self.assertDictEqual(filestate.comment(name, regex), ret)
 
                 with patch.dict(filestate.__salt__,
-                                {'file.search': MagicMock(side_effect=[False, True, False, True, True]),
+                                {'file.search': MagicMock(side_effect=[True, True, True]),
                                  'file.comment': mock_t,
                                  'file.comment_line': mock_t}):
                     with patch.dict(filestate.__opts__, {'test': True}):
@@ -1215,7 +1215,9 @@ class TestFileState(TestCase, LoaderModuleMockMixin):
 
             mock_t = MagicMock(return_value=True)
             mock_f = MagicMock(return_value=False)
-            mock = MagicMock(side_effect=[True, False, False, False, True, False,
+            mock = MagicMock(side_effect=[False, True,
+                                          False, False,
+                                          True,
                                           True, True])
             with patch.object(os.path, 'isabs', mock_f):
                 comt = ('Specified file {0} is not an absolute path'.format(name))

